docs/Components.md

```diff
@@ -26,4 +26,4 @@ In reading this specification, it is important to remember that NCA is concerned
 
 A Controller needs a control protocol to access Device control APIs. NCA's Control Model defines APIs abstractly, leaving their realizations in protocols to separate specifications.
 
-[IS-12 NMOS Control Protocol](https://specs.amwa.tv/is-12) defines a JSON-encoded Remote Procedure Call (RPC) protocol for NCA. All NCA compliant protocols share the same traits as defined by the [Control Model](Control%20Model.md).
+All NCA compliant protocols share the same traits as defined by the [Control Model](Control%20Model.md).
```

Comment:
Hmmm. I'm not sure about this sentence firstly for the use of trait, which could mean characteristic or perhaps software traits. And secondly by because it seems to suggest that the protocol behaviour is only bound by the Control Model as defined in the referenced document.  Straw man suggestion below:
```suggestion
All NCA compliant protocols implement the Control Model as described in [Control Model](Control%20Model.md) and defined in MS-05-02.
```